src/game/client/optionpreferences.cpp

```diff
@@ -490,36 +485,52 @@ bool OptionPreferences::Get_Firewall_Need_Refresh()
 
 uint32_t OptionPreferences::Get_LAN_IPAddress()
 {
+#ifndef THYME_STANDALONE
     // TODO needs IPEnumeration
     return Call_Method<uint32_t, OptionPreferences>(0x00462EA0, this);
+#else
+    return 0;
+#endif
 }
 
 void OptionPreferences::Set_LAN_IPAddress(AsciiString address)
 {
+#ifndef THYME_STANDALONE
     // TODO needs IPEnumeration
     return Call_Method<void, OptionPreferences, AsciiString>(0x004630E0, this, address);
+#endif
 }
 
 void OptionPreferences::Set_LAN_IPAddress(uint32_t address)
 {
+#ifndef THYME_STANDALONE
     // TODO needs IPEnumeration
     return Call_Method<void, OptionPreferences, uint32_t>(0x004632B0, this, address);
+#endif
 }
 
 uint32_t OptionPreferences::Get_Online_IPAddress()
 {
+#ifndef THYME_STANDALONE
     // TODO needs IPEnumeration
     return Call_Method<uint32_t, OptionPreferences>(0x004634B0, this);
+#else
+    return 0;
+#endif
 }
 
 void OptionPreferences::Set_Online_IPAddress(AsciiString address)
 {
+#ifndef THYME_STANDALONE
     // TODO needs IPEnumeration
     return Call_Method<void, OptionPreferences, AsciiString>(0x004636F0, this, address);
```

Comment:
Wouldn't it make more sense to simply have dummy `Call_Method` when `THYME_STANDALONE` is defined?

Reply:
Would I not still need an ifdef for the standalone return on none void functions though?

Reply:
Only if you return the non-default value.

Reply:
Good point. I'd have to rejig the includes as currently I conditionally include the header for Call_Method based on if the build is standalone or not as well.

Reply:
Up to you. In case of OpenRCT2 it proved useful, as I was able to repurpose such (similar) macro to only use the data section while `.test` was not used at all.